README.md

```diff
@@ -1,3 +1,141 @@
 # blinkmobile / angular-location [![npm](https://img.shields.io/npm/v/@blinkmobile/angular-location.svg?maxAge=2592000)](https://www.npmjs.com/package/@blinkmobile/angular-location) [![Travis CI Status](https://travis-ci.org/blinkmobile/angular-location.js.svg?branch=master)](https://travis-ci.org/blinkmobile/angular-location.js)
 
 Angular 1.x components for maps and geolocation
+
+
+## API
+
+
+### Configuration
+
+You must define a "googleMapsApiKey" property on the `$rootScope` with your API key.
```

Comment:
I definitely want to be writing idiomatic / conventional Angular code here.

I'm using a 3rd-party component called ngMap, and this is how _they_ do it: https://github.com/allenhwkim/angularjs-google-maps#lazy-loading-of-google-maps-javascript

I think making this a binding would be fine if there was only 1 of occurrence, but the service / constant / `$scope` approach seemed like a better choice for multiple uses. /shrug

I'll do a bit more reading around this subject.